src/app/(docs)/components/input/page.mdx

```diff
@@ -16,14 +16,18 @@ Some Key Features of Tag Input are:
 - Add keywords by pressing Enter or comma
 - Remove keywords by clicking the remove button (X icon) or pressing Backspace
 - Paste multiple keywords separated by commas, new lines, or tabs
+- Prevents duplicate keywords by using the unique prop
 
 <div className="mb-4"></div>
 <Preview>
-<div className="max-w-md w-full my-12">
-<TagInput/>
+<div className="max-w-md w-full my-12 flex flex-col gap-4">
+  <div className="text-lg font-semibold">Duplicate Keywords</div>
```

Comment:
remove the duplicate one along with the heading + also remove the heading for Unique Keywords aswell. 

Let's only show `<TagInput unique />` that makes more sense. 

Lmk once you are done with the changes, i will merge it then